manifests/params.pp

```diff
@@ -39,4 +39,10 @@
   String[1] $postgresql_evr_package = $katello::globals::postgresql_evr_package,
   Boolean $pulp2_support = $katello::globals::pulp2_support,
 ) inherits katello::globals {
+
```

Comment:
This empty line breaks CI once we update to the latest lint module. Please drop it.

Reply:
That still makes me sad :/ The empty line provides a nice break when reading the code.